src/Akeneo/Connectivity/Connection/back/Application/Webhook/Log/WebhookRequestLog.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Application\Webhook\Log;
+
+use Akeneo\Connectivity\Connection\Domain\Webhook\Client\WebhookRequest;
+
+/**
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class WebhookRequestLog
+{
+    /** @var WebhookRequest */
+    private $webhookRequest;
+
+    /** @var float */
+    private $startTime;
+
+    /** @var float */
+    private $endTime;
+
+    public function __construct(
+        WebhookRequest $webhookRequest,
+        float $startTime,
+        float $endTime = null
+    ) {
+        $this->webhookRequest = $webhookRequest;
+        $this->startTime = $startTime;
+        $this->endTime = $endTime;
+    }
```

Comment:
This is dead code, you don't use the $endTime in the constructor.
```suggestion
    public function __construct(
        WebhookRequest $webhookRequest,
        float $startTime
    ) {
        $this->webhookRequest = $webhookRequest;
        $this->startTime = $startTime;
    }
```